priv/perf/apps/load_test/lib/runner/smoke.ex

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Runner.Smoke do
```

Comment:
I introduce this to be a real smoke test when we have the automation load test setup on pipeline.
Since I would imagine the load test would be long running, having a simple smoke test to check whether the service is up can stop the long running job from even starting and make it clear on what is happening.

But I guess we might far away from my automation vision? Then this would become dead code then 😢

Reply:
Agree with @boolafish - this is the smoke test that checks everything is up and running